reflekt-plugin/src/main/kotlin/io/reflekt/plugin/analysis/processor/source/uses/FunctionUsesProcessor.kt

```diff
@@ -24,10 +25,15 @@ class FunctionUsesProcessor(override val binding: BindingContext, reflektInvokes
         return fileToUses
     }
 
-    override fun shouldRunOn(element: KtElement) = element.isPublicFunction && !element.isMainFunction
+    // TODO: how can we return the member functions??
```

Comment:
what exactly do you want to return?

Reply:
currently, we find only top-level functions, but we also can search for class member functions. But we have to handle them by another way since we can not just call them